docker-contributor/Dockerfile

```diff
@@ -31,6 +31,9 @@ RUN apt-get update \
     supervisor apache2-utils lsb-release \
     libcurl4-gnutls-dev libjsoncpp-dev libmagic-dev \
     enscript lpr ca-certificates less vim \
+    php-pear php-dev \
+    && pecl install pcov
+    && sed -i 's/;extension=xsl/extension=pcov.so/g' /etc/php/7.4/cli/php.ini
```

Comment:
Maybe better to write a file in /etc/php/7.4/cli/conf.d?
Then we don’t need a sed

Reply:
Is indeed a lot better. I'll change it.